governance/polls/Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021.md

```diff
@@ -0,0 +1,89 @@
+summary: This MIP set adds the Collateral Engineering Services Core Unit, CES-001
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp20-adding-collateral-engineering-services-core-unit-ces-001/9809
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-09-13T16:00:00
+end_date: 2021-09-27T16:00:00
+---
+# Inclusion Poll for Ratification Poll for Add Collateral Engineering Services Core Unit, CES-001 - September 13, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, September 13 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Add Collateral Engineering Services Core Unit, CES-001 being accepted and implemented in the Maker Protocol.
+
+### Add Collateral Engineering Services Core Unit
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Scaling Dai and decentralized collateral management are the biggest motivators for the Collateral Engineering Services (CES) Core Unit. The Maker protocol and community has an understanding and strong consensus that collateral is our top priority. We have done a great job since the Multi-collateral Dai launch in November 2019. Since then and using the resources available to us, we have designed and iterated on a collateral onboarding process that has served us well. Now, it is time to greatly improve upon our capabilities to reach one trillion Dai and beyond.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/597512747641658bc38c72bc17d1c84e43153c2b/MIP39/MIP39c2-Subproposals/MIP39c2-SP20.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP20)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp20-adding-collateral-engineering-services-core-unit-ces-001/9809)
+
+### Add Core Unit Budget, CES-001
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Building upon the theme of sustainability of the Collateral Engineering Services Core Unit (CES), the budget reflects support of the Mandates and business continuity in MIP39. Any elements of this budget reflecting a likeness to other Core Units is a principle of building on the shoulders of giants. Much appreciation is given to all the trailblazers.
```

Comment:
Will have to add a * at the end of line. Every new line will need this for markdown, you can use the preview option to see if your italics are working